Sources/FluentKit/Concurrency/Siblings+Concurrency.swift

```diff
@@ -2,57 +2,104 @@ import NIOCore
 
 public extension SiblingsProperty {
     
-    func load(on database: Database) async throws {
+    func load(on database: any Database) async throws {
```

Comment:
Perhaps we can also enable the flag in Package.swift ?

Reply:
That would require a secondary `Package@swift-5.8.swift` manifest and necessitate changes in almost every single file, I've been deliberately doing this gradually in most packages.

Reply:
Fine